service/observiq-otel-collector

```diff
@@ -0,0 +1,114 @@
+#!/bin/sh
+# observIQ OTEL daemon
+# chkconfig: 2345 99 05
+# description: observIQ's distribution of the OpenTelemetry collector
+# processname: observiq-otel-collector
+# pidfile: /var/run/observiq-otel-collector.pid
+
+# Source function library.
+if [ -e /etc/init.d/functions ]; then
+  STATUS=true
+  . /etc/init.d/functions
+fi
+
+if [ -e /lib/lsb/init-functions ]; then
+  PROC=true
+  . /lib/lsb/init-functions
+fi
+
+# Pull in sysconfig settings
+[ -f /etc/sysconfig/observiq-otel-collector ] && . /etc/sysconfig/observiq-otel-collector
```

Comment:
What would this file be used for if it existed, environment variables to configure the collector?

Reply:
For more information, this doc is good. Currently, we probably wouldn't be using this. However, in theory all the environment variables could be moved to this file and used that way.

https://docs.oracle.com/en/operating-systems/oracle-linux/6/admin/ol_about_kernparm.html